js/gamestate.js

```diff
@@ -198,6 +196,9 @@ class GameState {
 
   draw() {
     // Make a copy so that we can edit this card
+    if (!this.rng) {
```

Comment:
I don't think you should create a new rng here. It should be in game-start if anything